adal/src/androidTest/java/com/microsoft/aad/adal/AuthenticationContextTest.java

```diff
@@ -856,7 +856,7 @@ public void testRefreshTokenPositive() throws IOException, InterruptedException,
 
         final String response2 = "{\"id_token\":\""
                 + TEST_IDTOKEN
-                + "\",\"access_token\":\"TokenReturnsWithIdToken\",\"token_type\":\"Bearer\",\"expires_in\":\"10\",\"expires_on\":\"1368768616\",\"refresh_token\":\"refreshABC\",\"scope\":\"*\"}";
+                + "\",\"access_token\":\"TokenReturnsWithIdToken\",\"token_type\":\"Bearer\",\"expires_in\":\"3600\",\"expires_on\":\"1368768616\",\"refresh_token\":\"refreshABC\",\"scope\":\"*\"}";
```

Comment:
I'm not arguing that I prefer 10 seconds. if you change any data setting in existing tests, it's a red sign to me, I simply want to know why? Is it because some test are failing? So you change it to make it pass? If so, what cause the failure? Is it because of the new change? If it's a reasonable reason that you want to make the change, I will agree. But if it's I change it because it's easy for me to debug, I don't think I'll accept the answer. Your first answer "access token is expired in one hour" is a totally acceptable answer to me.